frontend/src/app/authentication/authenticated.component.html

```diff
@@ -10,7 +10,8 @@
     <span *ngIf="user.role === 'admin'">admin {{ user.adminUsername | AdminUserFormatterPipe }}</span>
     <span *ngIf="user.role ==='user'">{{ user.email }}</span>
     <nav class="user-link" id="authentication-nav" aria-label="Authentication">
-      <a id="view-applications" href="#" *ngIf="user.role !== 'admin'" routerLink="/user/applications"> View applications. | </a>
+      <a id="view-applications" href="#" *ngIf="specialUse && user.role === 'admin'" routerLink="/admin/applications"> View applications. | </a>
```

Comment:
since the visibility of this link is based on the value of `specialUse` maybe there should only be one here where the value of `routerLink` is determined by something in the component. Ex.

```
# My attempt at Angular templating...
<a id="view-applications" href="#" *ngIf="specialUse" [routerLink]="applicationsLink"> View applications. | </a>
```
where `this.applicationsLink` in `authenticated.component.ts` contains the correct value.